website/blog/2019-06-26-7.5.0.md

```diff
@@ -0,0 +1,250 @@
+
+This [proposal](https://github.com/tc39/proposal-pipeline-operator) has a few variants that are being thought out. By testing this plugin, you can help proposal authors gather feedback about how pipelines should work but know that refactoring will be required if the semantics change throughout the proposal process.
+
+Since version 7.3.0, Babel supported the ["Smart" variant](https://github.com/tc39/proposal-pipeline-operator/wiki#proposal-4-smart-mix) of the pipeline operator proposal, and the ["minimal" variant](https://github.com/tc39/proposal-pipeline-operator/wiki#proposal0-original-minimal-proposal) since 7.0. Thanks to the joint efforts of [Thiago Arrais](https://twitter.com/thiagoarrais) and [James DiGioia](https://twitter.com/JamesDiGioia), you can now also test the ["F#" variant](https://github.com/tc39/proposal-pipeline-operator/wiki#proposal-1-f-sharp-style-only-with-await)!
+
+> If you have thoughts or comments about this specific proposal variant, you can reach out to [@JamesDiGioia](https://twitter.com/JamesDiGioia).
+
+How is the "F#" variant different from the "Smart" one? It doesn't have the concept of "topic references" (`#`), instead you can easily use arrow functions. This has the advantage of being more similar to current JavaScript, at the cost of a slighly less concise syntax.
+
+<table class="danger-force-code-no-wrap">
+  <thead>
+    <tr>
+      <th>Current JavaScript</th>
+      <th>F# pipeline</th>
+      <th>Smart pipeline</th>
+    </tr>
+  </thead>
+  <tr>
+
+<!-- DO NOT indent this table, or the code won't be highlighted -->
+<td>
+
+```js
+let newScore = boundScore(
+  0,
+  100,
+  add(7, double(person.score))  
+);
+```
+</td>
+
+<td>
+
+```js
+let newScore = person.score
+  |> double
+  |> n => add(7, n)
+  |> n => boundScore(0, 100, n);
+
+```
+</td>
+
+<td>
+
+```js
+let newScore = person.score
+  |> double
+  |> add(7, #)
+  |> boundScore(0, 100, #);     
+
+```
+</td>
+
+</tr>
+</table>
+
+Althought the two proposals are not bound to each other and they are not being developed together, you can use F# pipelines alongside with partial application (supported since Babel 7.4.0):
+```javascript
+let newScore = person.score
+  |> double
+  |> add(7, ?)
+  |> boundScore(0, 100, ?);
+```
+
+Note that, while it may look the same as the "Smart" pipeline variant, the partial application proposal only supports `?` in function call parameters. This means that, for example, `person |> #.score` is a valid "Smart" pipeline whose F# equivalent must use an arrow function: `person |> p => p.score`.
+
+The F# pipeline operator is also different regarding how `await` is handled:
+
+<table class="danger-force-code-no-wrap">
+  <thead>
+    <tr>
+      <th>Current JavaScript</th>
+      <th>F# pipeline</th>
+      <th>Smart pipeline</th>
+    </tr>
+  </thead>
+  <tr>
+
+<!-- DO NOT indent this table, or the code won't be highlighted -->
+<td>
+
+```js
+let id = (
+  await (
+    await fetch(url)     
```

Comment:
Here there are some whitespaces at the end of the line to make it as wide as the other examples in the table